site/docs/src/json/identity-providers/links/pending-link-request-identityProviderLink-in-body.json

```diff
@@ -0,0 +1,6 @@
+{
+  "identityProviderLink":{
+    "pendingIdPLinkId":"elY3lKxvCQvsH_kfEMmBNLIl6TmydtCrNjHpFgVdxb8",
```

Comment:
I might rename the corresponding other file for this request with a `deprecated` prefix just to make it clearer.